src/core/bpf/allow_bind/meson.build

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+allow_bind_skel_h = custom_target(
+        'allow-bind-skel.h',
+        input : 'allow-bind.bpf.c',
+        output : 'allow-bind-skel.h',
```

Comment:
nit: is it possible to stick to the convention of having `.skel.h` ending (similar to `.bpf.c`)? It seems minor, but it's good to have a single convention.